grouper/fe/handlers/permission_view.py

```diff
@@ -39,5 +39,8 @@ def viewed_permission(
 
     def get(self, *args: Any, **kwargs: Any) -> None:
         name = self.get_path_argument("name")
+        permission_arg = self.get_argument("permission_arg", None)
```

Comment:
How about using just `argument` instead of `permission_arg`? Easier to guess if you're trying to manually edit the URL :D

Reply:
Done